File: Data Structures & Algorithms/missing-number/submission-1.java
class Solution {
    public int missingNumber(int[] nums) {
        HashSet<Integer> set = new HashSet<>();
        int res=0;
        for(int i:nums) res+=i;
        int gigaNigga =0;
        for(int i=0;i<=nums.length;i++) {
           gigaNigga+=i;
        }
        return gigaNigga-res;
    }
}

File: Data Structures & Algorithms/daily-temperatures/submission-1.py
class Solution:
    def dailyTemperatures(self, temperatures: List[int]) -> List[int]:
        stk=[]
        nums = temperatures
        res=[0]*len(nums)
        for i in range(len(nums)):   
            while stk and stk[-1][0]<nums[i]:
                ele , index = stk.pop()
                res[index]=i-index
            stk.append((nums[i],i))
        
        return res
        